l2-frontend/src/pages/ResultsDepartment.vue

```diff
@@ -43,11 +43,15 @@
               </div>
             </div>
             <div class="row">
-              <div class="col-xs-9"></div>
-              <div class="col-xs-3">
+              <div class="col-xs-5"></div>
+              <div class="col-xs-7">
                 <div class="btn btn-blue-nb" style="margin-bottom: 5px;margin-top: 15px;"
-                     @click="print">
-                  Печать
+                     @click="print($event), by_doc=false">
+                  По отделению
+                </div>
+                <div class="btn btn-blue-nb" style="margin-bottom: 5px;margin-top: 15px; margin-left: 20px;"
+                     @click="print($event), by_doc=true">
```

Comment:
Плохой подход.
Почему не
```suggestion
                     @click="print(true)">
```
Выше будет тогда print(false)
А print опишу в другом комменте